core/templates/dev/head/pages/exploration_player/AnswerFeedbackPairDirective.js

```diff
@@ -64,6 +64,9 @@ oppia.directive('answerFeedbackPair', [
                   $scope.data.learnerAnswer, interaction.id,
                   interaction.customizationArgs));
             }
+            if ($scope.data.learnerAnswer === 'I would like a hint.') {
+              return $scope.data.learnerAnswer;
+            }
```

Comment:
Without that the learner answer (on requesting a hint) will show an empty bubble instead of 'I would like a hint'.